### ISSUE TITLE
Merge into Media Plugin?

### ISSUE THREAD

**Issue by jamesmontemagno** [roycornelissen/GMImagePicker.Xamarin#63]
Thoughts on merging this into: https://github.com/jamesmontemagno/MediaPlugin ? I have an older multi-picker on iOS and could use a new updated one. I think that I am fine using your package as a dependency, just need the code to be written up.

**Comment by roycornelissen** [comment]
Hi @jamesmontemagno, would love to integrate it. Though I also think there remains value in keeping it a pure Xamarin iOS package. So I guess using it as a dependency would be the cleanest option.

I’d be happy to make adjustments to make integration easier though.

**Comment by jamesmontemagno** [comment]
I agree with that, i am thinking just taking your library as a dependency and swapping out my currently old implementation for multi-photo picker.

**Comment by jeffreymora** [comment]
This would be great, can you make this happen. I currently am using MediaPlugin and am looking to implement this package for the multi photo aspect but I'm having difficulty dealing with the PHAssets returned. I love how MediaPlugin returns a path that i can readily use to upload the file to a server.
